src/server/database/migrations/20170714185201-removeEmptyContent.js

```diff
@@ -0,0 +1,10 @@
+exports.up = async (r) => {
+  r.table('Project')
+    .filter((doc) => doc('content').match('"blocks":\\[]'))
```

Comment:
Chances are this regexp will be fine. I'm going to update it to be a little bit more robust. This pattern:

`"?blocks"?:\s*\\[\s*]`

Will make the quotes optional while also allowing zero or more characters of whitespace before the empty block (which also may contain zero or more characters of whitespace).